src/terrain/clients/apps/raw.clj

```diff
@@ -1393,13 +1393,19 @@
                 :body         body
                 :content-type :json})))
 
+(def workspace-for-user
+  (memoize (fn [params]
+             (-> (apps-url "workspaces")
+                 (client/get (disable-redirects {:query-params params
+                                                 :as           :json}))
+                 :body))))
+
 (defn bootstrap
   []
-  (:body
-   (client/get (apps-url "bootstrap")
-               (disable-redirects
-                {:query-params (secured-params)
-                 :as           :json}))))
+  (-> (apps-url "bootstrap")
+      (client/get (disable-redirects {:query-params (secured-params)
+                                      :as           :json}))
+      :body))
```

Comment:
Just confirming this in case I missed something, but this looks like pure rearrangement? It does look better the changed way, just leaving a note in case there's something more I should look at here.

Reply:
Correct. I was originally going to cache responses from the Bootstrap endpoint, but it turns out that web hooks are included in the response body. I decided against doing that, but I'd already refactored the function at that point. I thought I'd just leave the changes in place. 😁